fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -69,6 +72,15 @@ default List<String> listTableNames(String dbName) {
         return Lists.newArrayList();
     }
 
+    /**
+     * List all cached table names by caching hive metastore
+     *
+     * @return a set of string containing all cached table names.
+     */
+    default Set<HiveTableName> listCachedTableNames() {
```

Comment:
I think it's better to use CacheUpdateProcessor. this interface isn't common enough

Reply:
you could refer to the MetastoreEventsProcessor.